scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -542,23 +520,23 @@ task oncotate_m2 {
           fi
 
         ${default="/root/oncotator_venv/bin/oncotator" oncotator_exe} --db-dir onco_dbdir/ -c $HOME/tx_exact_uniprot_matches.AKT1_CRLF2_FGFR1.txt  \
-            -v ${m2_vcf} ${case_id}.maf.annotated hg19 -i VCF -o TCGAMAF --skip-no-alt --infer-onps --collapse-number-annotations --log_name oncotator.log \
+            -v ${m2_vcf} $case_id.maf.annotated hg19 -i VCF -o TCGAMAF --skip-no-alt --infer-onps --collapse-number-annotations --log_name oncotator.log \
             -a Center:${default="Unknown" sequencing_center} \
             -a source:${default="Unknown" sequence_source} \
-            -a normal_barcode:${default=" " control_id} \
-            -a tumor_barcode:${case_id} \
+            -a normal_barcode:$control_id \
+            -a tumor_barcode:$case_id \
             ${"--default_config " + default_config_file}
     >>>
 
     runtime {
         docker: "${oncotator_docker}"
-        memory: "3 GB"
+        memory: select_first([mem, 3]) + " GB"
         bootDiskSizeGb: 12
-        disks: "local-disk 100 HDD"
-        preemptible: "${preemptible_attempts}"
+        disks: "local-disk " + select_first([disk_space_gb, 100]) + " HDD"
+        preemptible: select_first([preemptible_attempts, 2])
     }
 
     output {
-        File oncotated_m2_maf="${case_id}.maf.annotated"
+        File oncotated_m2_maf=glob("*.maf.annotated")[0]
```

Comment:
These globs[0] worry me.... I see what you are trying to do.  It's a bit fragile, but I admit the risk is low.  

And I'm generally okay with passing around entity_ids.... in fact, it can be useful when working in a data model such as FireCloud

Reply:
done